libass/ass_render.c

```diff
@@ -459,13 +628,30 @@ static void blend_vector_clip(ASS_Renderer *render_priv,
     if (!render_priv->state.clip_drawing_text)
         return;
 
-    // Get mask from cache
-    BitmapHashKey key;
-    key.type = BITMAP_CLIP;
-    key.u.clip.scale = render_priv->state.clip_drawing_scale;
-    key.u.clip.text = render_priv->state.clip_drawing_text;
+    OutlineHashKey key;
+    key.type = OUTLINE_DRAWING;
+    key.u.drawing.text = render_priv->state.clip_drawing_text;
+    OutlineHashValue *outline =
+        ass_cache_get(render_priv->cache.outline_cache, &key, render_priv);
```

Comment:
Does the removal of `BITMAP_CLIP` actually help performance? Or just code clarity?

Reply:
It's mostly code clarity. Potentially it can save some processing in cases where the same drawing text is used as regular drawing and clip but such cases are quite rare.